Src/WitsmlExplorer.Frontend/components/Modals/__tests__/ReportModal.test.tsx

```diff
@@ -20,7 +21,8 @@ jest.mock("@equinor/eds-core-react", () => mockEdsCoreReact());
 
 jest.mock("services/jobService", () => {
   return {
-    getUserJobInfo: () => MOCK_JOB_INFO
+    getUserJobInfo: () => MOCK_JOB_INFO,
+    getReport: () => MOCK_REPORT
```

Comment:
Actually, i don't see why we need another mocked report. Can you use the REPORT instead? In the `Should show the report once the job has finished` test, that's the report we are checking values for anyways.